src/components/Apis.jsx

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios'
+
+const gameApi = axios.create({baseURL: 'https://fuzzyjess-board-games.herokuapp.com'})
+
+export const fetchReviews = () => {
```

Comment:
I might have been tempted here to have this return the reviews, seems a little more readable as right now it returns an object with a reviews key. Good work on extracting this out into a separate file already 👍

Reply:
Okay, makes sense - changed that :-)